src/main/steps/urls.ts

```diff
@@ -112,7 +112,8 @@ export const RESPONDENT_ADDRESS_HISTORY: PageLink = `${RESPONDENT}${ADDRESS_HIST
 /* Applicant-start */
 export const APPLICANT: PageLink = '/applicant';
 export const APPLICANT_TASK_LIST_URL: PageLink = '/applicant/task-list';
-
+export const APPLICANT_CA_MANAGE_ORDER_PDF: PageLink = '/';
+export const APPLICANT_DA_MANAGE_ORDER_PDF: PageLink = '/';
```

Comment:
Looks these URLs not being used anywhere.

Reply:
removed unused constants